app/scripts/controllers/transactions/pending-tx-tracker.js

```diff
@@ -211,7 +211,7 @@ export default class PendingTransactionTracker extends EventEmitter {
     const { hash: txHash, txParams: { nonce, from } } = txMeta
     const networkNextNonce = await this.query.getTransactionCount(from)
 
-    if (parseInt(nonce, 16) >= parseInt(networkNextNonce, 16)) {
+    if (parseInt(nonce, 16) >= parseInt(networkNextNonce) {
```

Comment:
I would at least appreciate the radix being made explicit here, so this doesn't come as a surprise to future readers. The fact that `networkNextNonce` is in base-10 seems fairly unexpected, considering that the Ethereum JSON-RPC protocol uses hex for everything.

As a general rule it's better to be explicit about the radix whenever possible (unless the input truly is unknown, and could be either decimal or hex or octal). In this case it seems especially useful to be explicit.

```suggestion
    if (parseInt(nonce, 16) >= parseInt(networkNextNonce, 10) {
```